web/ui/react-app/src/graph/Graph.tsx

```diff
@@ -123,6 +151,11 @@ class Graph extends PureComponent<GraphProps, GraphState> {
               <SeriesName labels={labels} format />
             </div>
           ))}
+          {chartData.length > 1 && (
+            <div className="pl-1 mt-1 text-muted" style={{ fontSize: 13 }}>
+              Click: toogle serie, CTRL + click: toogle multiple series
```

Comment:
Can we style this identically to the query stats?

"toogle" -> "toggle"
"serie" -> "series"